### PR TITLE
IGNITE-8110

### PR DESCRIPTION
**Pull request by antkr**
Fixed ms to nanos transformation.

### CODE REVIEW

File: modules/core/src/test/java/org/apache/ignite/internal/processors/cache/store/GridCacheWriteBehindStoreSelfTest.java [hunk 1]
```diff
@@ -106,6 +106,42 @@ public void testSimpleStoreWithoutCoalescing() throws Exception {
         testSimpleStore(false);
     }
 
+    /**
+     * Checks that write behind cache flush frequency was correctly adjusted to nanos
+     * Expecting putAllCnt to be less or equal than elapsed time divided by flush frequency
```
Comment (dspavlov) [inline modules/core/src/test/java/org/apache/ignite/internal/processors/cache/store/GridCacheWriteBehindStoreSelfTest.java]: Please use dot at the end of sentence

File: modules/core/src/test/java/org/apache/ignite/internal/processors/cache/store/GridCacheWriteBehindStoreSelfTest.java [hunk 2]
```diff
@@ -106,6 +106,42 @@ public void testSimpleStoreWithoutCoalescing() throws Exception {
...
+    public void testSimpleStoreFlushFrequencyWithoutCoalescing() throws Exception {
+        initStore(1, false);
+
+        long writeBehindFlushFrequencyNanos = FLUSH_FREQUENCY * 1000 * 1000;
+        int threshold = store.getWriteBehindStoreBatchSize() / 10;
+
+        try {
+            long start = System.nanoTime();
+
+            for (int i =0; i < threshold / 2; i++)
```
Comment (dspavlov) [inline modules/core/src/test/java/org/apache/ignite/internal/processors/cache/store/GridCacheWriteBehindStoreSelfTest.java]: seems code reformat (including automatic) will be useful here

File: modules/core/src/main/java/org/apache/ignite/internal/processors/cache/store/GridCacheWriteBehindStore.java [hunk 3]
```diff
@@ -895,7 +895,7 @@ private void wakeUp() {
         protected Thread thread;
 
         /** Cache flushing frequence in nanos. */
-        protected long cacheFlushFreqNanos = cacheFlushFreq * 1000;
+        protected long cacheFlushFreqNanos = cacheFlushFreq * 1000 * 1000;
```
Comment (dspavlov) [inline modules/core/src/main/java/org/apache/ignite/internal/processors/cache/store/GridCacheWriteBehindStore.java]: Just wondering, do we have some constant like NANOS_IN_MILLIS in Ignite?

Comment (antkr) [reply modules/core/src/main/java/org/apache/ignite/internal/processors/cache/store/GridCacheWriteBehindStore.java]: No such public constants are present in Ignite, we have private NANOS_IN_MICROSECOND in CacheMetricsImpl and NANOS_IN_SECOND in IntervalBasedMeasurement.

File: modules/core/src/test/java/org/apache/ignite/internal/processors/cache/store/GridCacheWriteBehindStoreSelfTest.java (hunk 1 above)
Comment (antkr) [reply modules/core/src/test/java/org/apache/ignite/internal/processors/cache/store/GridCacheWriteBehindStoreSelfTest.java]: Fixed.

File: modules/core/src/test/java/org/apache/ignite/internal/processors/cache/store/GridCacheWriteBehindStoreSelfTest.java (hunk 2 above)
Comment (antkr) [reply modules/core/src/test/java/org/apache/ignite/internal/processors/cache/store/GridCacheWriteBehindStoreSelfTest.java]: Fixed.

File: modules/core/src/main/java/org/apache/ignite/internal/processors/cache/store/GridCacheWriteBehindStore.java (hunk 3 above)
Comment (dspavlov) [reply modules/core/src/main/java/org/apache/ignite/internal/processors/cache/store/GridCacheWriteBehindStore.java]: May be in future we should create 1 place for such constants. Thank you for reply

**Pull request closed without merging**